share/completions/git.fish

```diff
@@ -1016,6 +1016,7 @@ complete -f -c git -n '__fish_git_using_command commit' -l fixup -d 'Fixup commi
 complete -f -c git -n '__fish_git_using_command commit' -l squash -d 'Squash commit to be used with rebase --autosquash'
 complete -c git -n '__fish_git_using_command commit' -l reset-author -d 'When amending, reset author of commit to the committer'
 complete -x -c git -n '__fish_git_using_command commit' -l author -d 'Override the commit author'
+complete -x -c git -n '__fish_git_using_command commit' -l cleanup -a 'strip whitespace verbatim scissors default' -d 'Cleanup mode'
```

Comment:
This description doesn't say anything.

Ideally we'd describe not just the option itself ("how to clean up message before committing") but also the options:

```fish
"strip\t'Leading/trailing whitespace/empty lines, #commentary'
 whitespace\t'Like strip but keep #commentary'
 verbatim\t'Do not change the message'
 scissors\t'Like whitespace but also remove after scissor lines'
 default\t'Like strip if the message is to be edited, whitespace otherwise'"
```

Reply:
Thank you for the suggestion @faho.